examples/inspection/plot_partial_dependence.py

```diff
@@ -213,3 +225,61 @@
 plt.subplots_adjust(top=0.9)
 
 plt.show()
```

Comment:
You should remove this as well.

Reply:
Fixed in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/d9b117cf009dfa8cf2b1f0c68763670e76adca52